### ISSUE TITLE
Use subset of features till Installation error (OpenTargets compatibility) is resoved

### ISSUE THREAD

**Issue by titoghose** [sdecesco/targetDB#11]
Hello, there seems to be an issue with opentargets when trying to install the package (as is mentioned in the README as well). I was wondering if it is possible to get an older working version, or any other fix to this problem?

As of now, if a fix is not possible but it is possible to use a subset of features such as obtaining the "safety score", that does not depend on OpenTargets, that would also be amazing.

**Comment by sdecesco** [comment]
Hi, 

A fix is in the works. OpenTargets changed it's API to GraphQL therefore some code adaptation is needed

**Comment by titoghose** [comment]
Hello,
Thanks very much!

I have applied a temporary fix to a local version of your code that disables the OpenTargets functionality and it works now. I also had to make some other minor changes in some of the Pandas functionality (reading of "xlsx" files is not supported by the excel backend and it only supports "xls" files now; multi-indexing was producing some odd errors in a section of the code). 

Looking forward to the permanent fix!
